server/api/employee/employee.model.js

```diff
@@ -0,0 +1,78 @@
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+var employeeSchema = new mongoose.Schema({
+	
+	first_name: {
+		type: String,
+		required: true
+	},
+	
+	last_name: {
+		type: String,
+		required: true
+	},
+	
+	client: {
+		type: String,
+		required: true
+	},
+	
+	skill: {
+		type: String,
```

Comment:
employee can have knowledge on other language skills ,so skill should be array instead of string.